migration/v4/model/migrationwave.go

```diff
@@ -0,0 +1,13 @@
+package model
+
+import "time"
+
+type MigrationWave struct {
```

Comment:
New model. Each Application may be assigned to a single MigrationWave. Deleting a wave should null the MigrationWaveID on the Application model.